docs/manual.rst

```diff
@@ -1524,13 +1524,13 @@ derived from set-theoretic analysis.
 
   The styles of caps are specified by integer values: 1 (round), 2 (flat),
   3 (square). These values are also enumerated by the object
-  :class:`shapely.geometry.CAP_STYLE` (see below).
+  :class:`shapely.constructive.BufferCapStyles` (see below).
```

Comment:
```suggestion
  :class:`shapely.BufferCapStyles` (see below).
```

We import it in the top-level namespace, so we can refer to it like that here as well

Reply:
and the same for the few other occurrences below